scripts/steps/install_dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -x
+
+# Generates a new package manifest and resolves dependencies.
+# When run on CI, set_references.sh should be run before this script.
+
+# Ensure release/ is present
+mkdir -p release
+
+# Generate Package.swift with branches set, if needed.
+swift ./scripts/generatePackageSwift.swift > Package.swift
+
+# Dump the manifest to the console
+cat Package.swift
```

Comment:
I don't see service packages in the manifest here https://github.com/awslabs/aws-sdk-swift/actions/runs/3347731124/jobs/5546034658#step:7:2947